tests/test_tag.py

```diff
@@ -26,13 +26,76 @@
 @pytest.mark.usefixtures(conftest.app_context.__name__)
 def test_create_tag(user: UserClient):
     """Creates a tag specifying a custom organization."""
+    # import pdb; pdb.set_trace()
```

Comment:
Debugging lines could be cleaned up but it is not imperative in the tests either. @cayop As you prefer or think is best.